easybuild/toolchains/compiler/craype.py

```diff
@@ -59,12 +59,12 @@ class CrayPECompiler(Compiler):
     COMPILER_FAMILY = None
 
     COMPILER_UNIQUE_OPTS = {
-        'dynamic': (False, "Generate dynamically linked executable"),
+        'dynamic': (True, "Generate dynamically linked executable"),
         'mpich-mt': (False, "Directs the driver to link in an alternate version of the Cray-MPICH library which \
                              provides fine-grained multi-threading support to applications that perform \
                              MPI operations within threaded regions."),
         'optarch': (False, "Enable architecture optimizations"),
-        'verbose': (True, "Verbose output"),
+        'verbose': (False, "Verbose output"),
```

Comment:
The argument was that verbose was needed while TC was experimental, but now we could make it optional.

But you definitely have a point here since we lose the actual command line parameters behind the wrappers. Wrappers arguments are usually minimal and if we disable verbose I suppose we won't see on the logs all the includes, links and so on... 

@pforai what do you think?